populateDb/index.js

```diff
@@ -0,0 +1,36 @@
+const { DynamoDBClient, BatchWriteItemCommand } = require( "@aws-sdk/client-dynamodb");
+const ProductsSource = require("../productService/products.json");
+
+(async () => {
```

Comment:
It's not necessary to call it as IIFE, you can just require this module and it would be executed